migrations/versions/0467_add_annual_limits_data.py

```diff
@@ -0,0 +1,29 @@
+"""
+Revision ID: 0464_add_annual_limits_data
+Revises: 0463_add_annual_limit_column
+Create Date: 2024-10-31 13:32:00
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects.postgresql import UUID
+
+revision = "0464_add_annual_limits_data"
+down_revision = "0463_add_annual_limit_column"
+
+
+def upgrade():
+    op.create_table(
+        "annual_limits_data",
+        sa.Column("service_id", UUID, nullable=False),
+        sa.Column("time_period", sa.VARCHAR, nullable=False),
```

Comment:
Curious what the values for `time_period` will look like, something like `2024-Q1`?

Reply:
ya something like that